src/main/proto/mesos-utils.proto

```diff
@@ -0,0 +1,17 @@
+package mesosphere.mesos.protos;
+
+option java_package = "mesosphere.mesos.protos";
+option java_outer_classname = "StateProtos";
+
+// Describes a state entry, a versioned (via a UUID) key/value pair.
+// Copied from libmesos (src/messages/state.pro) to ensure that ZookeeperDirectState is
```

Comment:
You mean from the mesos jar? It's not included in there. I could leave instructions for users to copy the file itself from ./src/messages/state.proto, is that what you mean?